transpiler/main/c_code_gen.c

```diff
@@ -230,6 +231,9 @@ void transpileStmt(struct Stmt* s, struct Ctx* ctx){
 		transpileAssignStmt(s->m5, ctx);
 		fprintf(ctx->file, ";");
 		
+	}else if(s->m6 != NULL){
+		transpileBreakStmt(s->m6, ctx);
+		
 	}else{
 		printf("Error in transpileStmt\n");
```

Comment:
ctx and stmt not free'd

Reply:
(not reall) fixed in 3ae5ceebf663834cc74fb46be03e07e5f536da85